worker-jobtracking/src/test/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerFactoryTest.java

```diff
@@ -88,10 +90,32 @@ public void testProxiedCompleteTask() throws Exception {
     @Test
     public void testTrackingReportTask() throws Exception
     {
+        final WorkerTask wtd = Mockito.mock(WorkerTask.class);
```

Comment:
So far as I can see the changes to this test are not related to SCMOD-11919.  Move them to a separate PR where they can be considered on their own merits.  If there is an issue with them we don't want that holding up the SCMOD-11919 changes.

https://github.com/salemove/github-review-helper/blob/master/doc/rules.md

Reply:
removed and stored on a separate branch